packages/application-shell/src/components/navbar/use-navbar-state-manager.ts

```diff
@@ -31,12 +40,12 @@ type State = {
   isMenuOpen: boolean;
 };
 type Action =
-  | { type: 'setActiveItemIndex'; payload: string }
-  | { type: 'unsetActiveItemIndex' }
-  | { type: 'setIsExpanderVisible' }
-  | { type: 'toggleIsMenuOpen' }
-  | { type: 'setIsMenuOpenAndMakeExpanderVisible'; payload: boolean }
-  | { type: 'reset' };
+  | { type: ActionTypes.setActiveItemIndex; payload: string }
```

Comment:
If I understood enums right, 
it's enough that this is `{ type: ActionTypes }`, but this is more explicit so I keep it this way